protocol/src/main/resources/message.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://tbdex.dev/json-schemas/message.schema.json",
+  "$id": "https://tbdex.dev/message.schema.json",
```

Comment:
`networknt` seems to be having trouble resolving `$ref`s -- particularly `https://tbdex.dev/definitions.json`. If I update this `$id` back to `https://tbdex.dev/json-schemas/message.schema.json`, `networknt` is able to find the definitions without error, even though it's looking for definitions in `definitions.json` which has `$id: "https://tbdex.dev/definitions.json"`.

These same JSON schemas seem to be working in tbdex-js. It's an issue of `$ref` resolution and even after looking through `networknt` code I'm a little stumped.

Reply:
Branch: https://github.com/TBD54566975/tbdex-kt/compare/submodules-madness?expand=1

This one's a doozy. Seems like `networknt` is actually resolving the URL `https://tbdex.dev/json-schemas/definitions.json`. I did not realize it would even attempt that. `MessagesVectorTest.kt` works online. I tried disconnecting my laptop from wifi and the test failed. How do I get `networknt` to look at local JSON schemas? I guess it's time for me to read some documentation.

Reply:
Fixed by using `SchemaValidatorsConfig#addUriTranslator`. Docs [here](https://github.com/networknt/json-schema-validator/blob/master/doc/schema-map.md)

Reply:
@diehuxx could you explain why `/json-schemas/` part of the uri in the `$id` were removed? how is  the uriTranslator related to this?